crates/bevy_pbr/src/render/mesh_view_bindings.wgsl

```diff
@@ -44,9 +44,10 @@
 @group(0) @binding(16) var dt_lut_sampler: sampler;
 
 #ifdef MULTISAMPLED
-
 #ifdef DEPTH_PREPASS
+#ifndef WEBGL2
```

Comment:
Why was this WEBGL2 check added? I might have missed this conversation. Also, does it cause problems if this binding is defined on the CPU, but not used in the shader? I assume it's fine.

Reply:
Hmm. This is actually a leftover of me resolving too many merge conflicts and forgetting to clean this up in a wholistic way. At some point I added an `#ifndef` here in my branch, but it now is redundant since `DEPTH_PREPASS` will never be defined under WebGL anyway.

Let me fix this.